posthog/hogql/query.py

```diff
@@ -132,6 +132,7 @@ def execute_hogql_query(
             context=clickhouse_context,
             dialect="clickhouse",
             settings=settings or HogQLGlobalSettings(),
+            timeout=600 if in_export_context else None,
```

Comment:
I guess I wanted to be more explicit and give the option to override the max_execution_time... But yeah, since settings already contains it, and you pass it in here, it can be just set there.